### ISSUE TITLE
Radio doesn't work

### ISSUE THREAD

**Issue by vfsfitvnm** [vfsfitvnm/ViMusic#186]
It looks like youtube started checking the request payload - data must be present and valid. So far this only affects the `/next` endpoint - which means the radio won't suggest songs anymore.

**Comment by vfsfitvnm** [comment]
Nah, ok, false alarm; it's true they changed something - but it's an easy fix

**Comment by fardeen619** [comment]
What actually radio feature does?

**Comment by HemanthJabalpuri** [comment]
> What actually radio feature does?

If I am not wrong, when you use 'Start Radio', it will fill playlist with similar songs to the one that is playing.
